README.md

```diff
@@ -56,7 +56,7 @@ Launch with:
 
 ### Running in a remote installation
 
-You need some configuration before starting Jupyter notebook remotely. See [Running a notebook server](https://jupyter-notebook.readthedocs.io/en/stable/public_server.html).
```

Comment:
Is it possible to point to older version of Jupyter notebook docs instead? The jupyter-server options will not work with the notebook server.